llvm/test/MC/WebAssembly/annotations-typecheck.s

```diff
@@ -0,0 +1,11 @@
+# RUN: not llvm-mc -triple=wasm32-unknown-unknown < %s | FileCheck %s
```

Comment:
so this means that the assembly as a whole still fails, right?

Reply:
It fails because type checking fails. But at least `InstPrinter` and its annotation printer is not printing confusing messages.